package.json

```diff
@@ -16,9 +16,9 @@
     "fast-glob": "3.2.4",
     "formidable": "1.2.2",
     "gm": "1.23.1",
-    "grunt": "1.3.0",
+    "grunt": "^1.3.0",
     "grunt-check-modules": "1.1.0",
-    "grunt-cli": "1.3.2",
+    "grunt-cli": "^1.3.2",
```

Comment:
`grunt*` updates shouldn't be needed

Reply:
Thank you, I've reverted this for my next commit.

Reply:
This is still using the caret, please also revert that.